src/Aspire.Hosting/RabbitMQ/RabbitMQBuilderExtensions.cs

```diff
@@ -26,7 +26,6 @@ public static IResourceBuilder<RabbitMQContainerResource> AddRabbitMQContainer(t
         var rabbitMq = new RabbitMQContainerResource(name, password);
         return builder.AddResource(rabbitMq)
                        .WithAnnotation(new EndpointAnnotation(ProtocolType.Tcp, port: port, containerPort: 5672))
-                       .WithAnnotation(new EndpointAnnotation(ProtocolType.Tcp, uriScheme: "http", name: "management", port: null, containerPort: 15672))
```

Comment:
No more management UI?

Reply:
No. I think we need to move it to an extension anyway (I think this is something we discussed).

I was also looking at changing the container image but there isn't a generic 3.x tag that doesn't include the management stuff.

We need to have a think about the way we handle container image version defaults in aspire.

Reply:
File an issue and change the image to the one that doesn't have the UI

Reply:
I ended up spotting the `"3"` tagged image. I just missed as I was scanning through the set of tags. I'll still file a work item for discussing how we want to handle container image version management.

Reply:
https://github.com/dotnet/aspire/issues/1997